packages/appstore/src/constants/tour-steps-config.tsx

```diff
@@ -0,0 +1,89 @@
+import { Step, Styles, Locale } from 'react-joyride';
+import React from 'react';
+import { Text, Button } from '@deriv/components';
+import { localize } from '@deriv/translations';
+
+export const tour_step_config: Step[] = [
+    {
+        title: (
+            <Text as='p' size='s' weight='bold' color='brand-red-coral'>
```

Comment:
size='s' is the default value of Text component, you can remove it from here.

Reply:
Alrigghtss